### DISCUSSION TITLE
Tailwind3: cannot use flex:col and then md:flex row?

### DISCUSSION THREAD

**Discussion by Github010101011** [tailwindlabs/tailwindcss#6577] [Help]
I want to use flex-col on mobile and transition to md:flex-row on md screens and larger.

it worked on tailwind2. But it doesn’t seem to work any longer. It will stay in flex col even when the screen size is md or larger

<div class=“flex flex-col md:flex-row”>
</div>

**Comment by lukewarlow** [comment]
https://github.com/tailwindlabs/tailwindcss/discussions/6579 - responded on your duplicate dicussion
